lib/puppet/pops/types/types.rb

```diff
@@ -1272,6 +1488,7 @@ def is_the_empty_hash?
       end
 
       DEFAULT = PHashType.new(nil, nil)
+      TUPLE_SIZE = PIntegerType.new(2,2)
```

Comment:
The size of a TUPLE in a key pair. What other tuples are there :-). Seriously. I'll change it to KEY_PAIR_TUPLE_SIZE.